cmd/keeper/cmd/keeper.go

```diff
@@ -477,6 +481,8 @@ func NewPostgresKeeper(cfg *config, end chan error) (*PostgresKeeper, error) {
 
 		dataDir: dataDir,
 
+		disableAlterSystem: cfg.disableAlterSystem,
```

Comment:
Pull this line down to just below the `requestTimeout` please!

Reply:
ok, done just right now